### PR TITLE
feat(dynamic-faceting): add guide using QR in RIS

### PR DESCRIPTION
**Pull request by Haroenv**

### CODE REVIEW

File: dynamic-faceting-qr/react-instantsearch/README.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        <Widget attribute={attribute} key={attribute} />
+      </Panel>
+    ))
+  }
+</DynamicFacets>
+```
+
+This gives us a final result of:
+
+[![screenshot](https://codesandbox.io/api/v1/sandboxes/2x0pvzvr8p/screenshot.png)](https://codesandbox.io/s/github/algolia/dynamic-faceting-guide)
```
Comment (Haroenv) [inline dynamic-faceting-qr/react-instantsearch/README.md]: still need to change these links once merged

File: dynamic-faceting-qr/react-instantsearch/README.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+This is exactly like normally, but this time you also keep track of which facets are possible. Here I've started from the "ecommerce" dataset provided, and indexed it like in [indexing.js](./indexing/index.js) (run `yarn index` to index to your app).
+
+then we decide which queries (no typo-tolerance) you want these facets to show up. For example:
+
+```js
+const dynamicFacets = [
+  {
+    query: 'iphone',
+    facets: [
+      { attribute: 'category', widgetName: 'RefinementList' },
```
[the remark this answers is not in the document]
Comment (samouss) [reply dynamic-faceting-qr/react-instantsearch/README.md]: Not only:

```js
// From the guide
categories, type, brand, price

// From the code
categories, type, manufacturer, salePrice
```

File: dynamic-faceting-qr/react-instantsearch/README.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+  .then(console.log);
+```
+
+These query rules can also be added via the dashboard. make sure to choose the same options as the ones added here.
+
+## Dynamically displaying facets
+
+In our query rules we made four possible dynamic facets: categories, type, brand, price. We now want to display these in a React InstantSearch app. First let's create a start:
+
+```jsx
```
[the remark this answers is not in the document]
Comment (Haroenv) [reply dynamic-faceting-qr/react-instantsearch/README.md]: funnily this is only in the diff, but changed it anyway

**Review by mthuret — CHANGES_REQUESTED** [review]
Hey @Haroenv, thanks for this PR! 
Can you modify the Readme to try to keep the same format as the other labs? See https://github.com/algolia/instantsearch-labs/tree/master/autocomplete%2BdelayedHits/instantsearch.js or https://github.com/algolia/instantsearch-labs/tree/master/instantsearch-relevance-widget/instantsearch.js 

Thanks in advance!

**Pull request closed without merging**